docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3.5-alpine
+
```

Comment:
I prefer to use Debian as the base system. We are using now debian:stretch-slim other docker images. I share the thoughts in: https://www.turnkeylinux.org/blog/alpine-vs-debian.

Reply:
Ack, I'll change that, I don't have a strong opinion on this excepted that the image size can be very problematic when we have to deal with a lot of different images which I tend to use alpine

Reply:
Cool. IMHO once this is changed I would merge the PR and start playing with it and if needed, evolve it. Nice work @olblak!